CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 # Changelog
 
-## Unreleased
-
 ## 0.1.0 (2018-07-27)
 
 Initial release.
+
+## 0.2.0RC *current master*
```

Comment:
Typically, you'd record changes in a changelog in reverse order, so latest release is on top. 

Also, any changes on current master that have not been released yet, are located under the `Unreleased` headline.

These recommendations are based on https://keepachangelog.com/en/1.0.0/, which is quite common ...